app/assets/javascripts/app/detail_notes.coffee

```diff
@@ -0,0 +1,12 @@
+$ ->
+  hiddenText = "More"
+  openText = "Less"
+
+  $('.js--toggleNote').on 'click', (e) ->
+    e.preventDefault()
+    $parent = $(@).parent()
+    text = $(@).text()
+    $parent.find(".js--note").toggle()
+    $parent.find(".js--truncatedNote").toggle()
+    $(@).text if text == hiddenText then openText else hiddenText
+    return
```

Comment:
Just curious, what does the bare `return` do in this case vs excluding it (like is common in ruby)?